src/ft2font.cpp

```diff
@@ -721,8 +706,7 @@ FT_UInt FT2Font::get_char_index(FT_ULong charcode, bool fallback = false)
         ft_object = this;
     }
 
-    // historically, get_char_index never raises a warning
-    return ft_get_char_index_or_warn(ft_object->get_face(), charcode, false);
+    return  FT_Get_Char_Index(ft_object->get_face(), charcode);
```

Comment:
Extra space
```suggestion
    return FT_Get_Char_Index(ft_object->get_face(), charcode);
```

Reply:
There are many, but unfortunately they have a ton of options, and I'm not sure if there's an existing setup for PEP7-ish style.